Quasardb/Native/qdb_exp_batch.cs

```diff
@@ -0,0 +1,109 @@
+using System.Runtime.InteropServices;
+
+// ReSharper disable BuiltInTypeReferenceStyle
+// ReSharper disable InconsistentNaming
+
+using qdb_size_t = System.UIntPtr;
+
+namespace Quasardb.Native
+{
+    internal enum qdb_exp_batch_push_mode : int
```

Comment:
Do these need to match an enum on the native C API side as well? If so, would be good to add a comment indicating this.

Reply:
done.